ui/cockpit/src/modules/HOC/withUser.js

```diff
@@ -52,6 +52,11 @@ export function UserProvider({ children }) {
       setUser(user);
       return user;
     } catch (err) {
+      if (err.status === 404) {
+        window.location.hash = "#/login";
+
+      }
+
```

Comment:
Adjust styling:
```suggestion
      if (err.status === 404) {
        window.location.hash = "#/login";
      }
```

Another thing to consider: we use `window.location.href` a bit above, maybe we want to stay consistent. However, this will break in the development setup, because the base tag is hard-coded to the default engine in the dev setup.